src/UploadHandler.php

```diff
@@ -1335,7 +1335,7 @@ public function post($print_response = true)
                 foreach ($upload['tmp_name'] as $index => $value) {
                     $files[] = $this->handle_file_upload(
                         $upload['tmp_name'][$index],
-                        $file_name ? $file_name : $upload['name'][$index],
+                        $file_name ? $file_name : strip_tags($upload['name'][$index]),
```

Comment:
I am not sure it will cover all possible cases. I guess it could be possible to inject a malicious value that does not contains HTML tags.

Reply:
To be more precise, I think there is a lack of special chars encoding in the `Html::uploadedFiles()` method.